src/RA_Dlg_MemBookmark.cpp

```diff
@@ -536,38 +536,33 @@ void Dlg_MemBookmark::SetupColumns(HWND hList)
 
     //	Remove all data.
     ListView_DeleteAllItems(hList);
+
+    auto idx{ 0 };
+    for (auto& sColTitle : ra::COLUMN_TITLE)
     {
-        auto lplvColumn{ std::make_unique<LV_COLUMN>() };
-        auto idx{ 0 };
-        for (auto& sColTitle : ra::COLUMN_TITLE)
+        ra::tstring tszText{ sColTitle };
+        LV_COLUMN col
         {
-            lplvColumn->mask       = LVCF_TEXT | LVCF_WIDTH | LVCF_SUBITEM | LVCF_FMT;
-            lplvColumn->fmt        = LVCFMT_CENTER | LVCFMT_FIXED_WIDTH;
-            lplvColumn->cx         = ra::COLUMN_WIDTH.at(idx);
-
-            // NB: The column title has to cached before-hand or we won't see it. -SBS
-            ra::tstring tszText{ sColTitle };
-            lplvColumn->pszText    = tszText.data();
-            lplvColumn->cchTextMax = 255;
-            lplvColumn->iSubItem   = idx;
-
-            if (idx == (ra::COLUMN_TITLE.size() - 1))
-                lplvColumn->fmt |= LVCFMT_FILL;
-
-            ListView_InsertColumn(hList, idx, lplvColumn.get());
-            idx++;
-        }
+            col.mask       = LVCF_TEXT | LVCF_WIDTH | LVCF_SUBITEM | LVCF_FMT,
+            col.fmt        = LVCFMT_CENTER | LVCFMT_FIXED_WIDTH,
+            col.cx         = ra::COLUMN_WIDTH.at(idx),
+            col.pszText    = tszText.data(),
+            col.cchTextMax = 255,
+            col.iSubItem   = idx
+        };
```

Comment:
Made it stack-based instead, the rest of the params aren't pointers so they get initilized to 0 automatically. Hopefully this looks clear.